### PR TITLE
Hotfix for 1.2

### PR DESCRIPTION
**Pull request by fanlei**
1. Fix the magic method name for unset()
2. Remove const MIN_CAPACITY assignment in Queue
3. Assign the current stamp to the instance copy
4. Fix documentation content.

### CODE REVIEW

File: src/Hashable.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ function hash();
      * Determines if two objects should be considered equal. Both objects will
      * be instances of the same class but may not be the same instance.
      *
-     * @param $obj An instance of the same class to compare to.
+     * @param $obj - An instance of the same class to compare to.
```
Comment (rtheunissen) [inline src/Hashable.php]: Why this `-`?

Comment (fanlei) [reply src/Hashable.php]: There is no type after @param so IDE such as Eclipse PDT would raise a syntax error in the doc block. However, I confess I was not sure what type to put there at the time when I was trying to fix it.
I checked further now and supposed $obj could be everything, according to https://github.com/php-ds/ext-ds/issues/140.  Do you think it's better to be 'mixed' there instead?

**Comment by fanlei** [comment]
Thank you for reviewing. Yes, I should keep it consistent.

**Comment by rtheunissen** [comment]
The tests and versioning is a bit of a mess it seems - haven't looked at this in a while. I'd like to consolidate all the components as part of a 1.3.0 release.

**Pull request merged**